lib/llm/src/preprocessor.rs

```diff
@@ -1,17 +1,5 @@
 // SPDX-FileCopyrightText: Copyright (c) 2024-2025 NVIDIA CORPORATION & AFFILIATES. All rights reserved.
 // SPDX-License-Identifier: Apache-2.0
-//
```

Comment:
@dmitry-tokarev-nv  can you confirm? 

I remember we initially wanted to keep it terse but later added the additional